src/styles/styles.scss

```diff
@@ -4,6 +4,7 @@
     "0.settings",
     "1.typography",
     "2.color",
+    "sprites",
```

Comment:
Yeah that's why I'm not a fan of numbering files or folders :)
But I can leave with this for now.